baseapp/baseapp.go

```diff
@@ -200,6 +200,8 @@ func (app *BaseApp) InitChain(req abci.RequestInitChain) (res abci.ResponseInitC
 	// NOTE: we're writing to the cms directly, without a CacheWrap
 	ctx := sdk.NewContext(app.cms, abci.Header{}, false, nil)
 
+	// TODO: d00d, tendermint sends nil, until that is fixed, we gotta
+	// read in the genesis file ourselves here
```

Comment:
Maybe remove or modify this comment - we currently read this in directly to initChainer during basecoin initialization

Reply:
No we do not. It panics on nil right now

Reply:
Fixed in last commit